### PR TITLE
Prisma in your stack: "Fullstack"

### PR DESCRIPTION
**Pull request by ruheni**
- add fullstack page
- update rest & graphql page
- rename is-prisma-an-orm page

### CODE REVIEW

File: content/200-concepts/050-overview/300-prisma-in-your-stack/02-graphql.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ metaDescription: 'This page gives explains how to build GraphQL servers with Pri
 
 [GraphQL](https://graphql.org/) is a query language for APIs. It is often used as an alternative to RESTful APIs, but can also be used as an additional "gateway" layer on top of existing RESTful services.
 
-With Prisma, you can build GraphQL servers that connect to a database. Prisma is completely agnostic to the GraphQL tools you use. When building a GraphQL server, you can combine Prisma with tools like Apollo Server, `express-graphql`, TypeGraphQL, GraphQL.js, or pretty much any tool or library that you're using in your GraphQL server setup.
```
Comment (ruheni) [inline content/200-concepts/050-overview/300-prisma-in-your-stack/02-graphql.md:11]: `express-graphql` has been deprecated.

File: content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+title: 'Fullstack'
+metaTitle: 'Building fullstack applications with Prisma'
+metaDescription: 'This page gives explains how to build fullstack applications with Prisma. It shows how Prisma fits in with fullstack frameworks and provides practical examples'
+---
+
+<TopBlock>
+
+Fullstack frameworks, such as Next.js and Sveltekit, blur the lines between the server and the client. These frameworks also provide different patterns for fetching and mutating data on the server.
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx]: :pig-angry:

```suggestion
Fullstack frameworks, such as Next.js, Remix or SvelteKit, blur the lines between the server and the client. These frameworks also provide different patterns for fetching and mutating data on the server.
```

File: content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+</TopBlock>
+
+## Supported frameworks
+
+Here's a non-exhaustive list of frameworks and libraries you can use with Prisma:
+
+- [Next.js](https://nextjs.org/)
+- [Remix](https://remix.run)
+- [Sveltekit](https://kit.svelte.dev/)
+- [Nuxt](https://nuxt.com/)
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx:22]: How about adding Redwood and t3 (or tRPC) to this bullet list as well?

File: content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
+}
+```
+
+You can now implement the logic for querying your database using [Prisma Client API](/concepts/components/prisma-client) inside `getServerSideProps`, `getStaticProps`, API routes, or using API libraries such as [tRPC](https://trpc.io/) and [GraphQL](https://graphql.org/).
+
+### `getServerSideProps`
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx]: this doesn't get rendered in the right font size:

image

- you can the `<inlinecode>` html tag in headings instead, here is an example in the [reference](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findfirst)

File: content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+```ts
+// (in /pages/index.tsx)
+// You can use `getStaticProps` in place of `getServerSideProps`
+export const getServerSideProps = async () => {
```
Comment (nikolasburk) [inline content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx]:
```suggestion
// (in /pages/index.tsx)

// Alternatively, you can use `getStaticProps`
// in place of `getServerSideProps`.
export const getServerSideProps = async () => {
```

**Review by nikolasburk — DISMISSED** [review]
This looks great! Left a few small comments but this is a helpful new page in the docs 👌

File: content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx (hunk 3 above)
Comment (ruheni) [inline content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx:22]:
```suggestion
- [Nuxt](https://nuxt.com/)
- [Redwood](https://redwoodjs.com/)
- [t3 stack — using tRPC](https://github.com/icflorescu/trpc-sveltekit)
```

File: content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
+}
+```
+
+You can now implement the logic for querying your database using [Prisma Client API](/concepts/components/prisma-client) inside `getServerSideProps`, `getStaticProps`, API routes, or using API libraries such as [tRPC](https://trpc.io/) and [GraphQL](https://graphql.org/).
+
+### `getServerSideProps`
```
Comment (ruheni) [inline content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx]:
```suggestion
### <inlinecode> getServerSideProps</inlinecode>
```

**Pull request merged**